Sources/StreamChatUI/ChatMessageList/Attachments/GiphyAttachmentViewInjector.swift

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+public typealias GiphyAttachmentViewInjector = _GalleryAttachmentViewInjector<NoExtraData>
+
+public class _GiphyAttachmentViewInjector<ExtraData: ExtraDataTypes>: _AttachmentViewInjector<ExtraData> {
```

Comment:
I've just realized we don't show attachment actions.

With @VojtaStavik we were discussing, that action view could be part of `ChatMessageContentView`. If there's a chance multiple attachments will have actions - that makes sense. If it's only giphy, then it would be probably better to keep actions UI encapsulated in the dedicated giphy injector.